File: beginner/python/1046.py
valores = map(int, input().split())

horaIni, horaFin = valores
cont = 0

while True:
    horaIni+=1
    cont+=1
    
    if(horaIni == 24):
        horaIni = 0
    
    if(horaIni == horaFin):
        break
    
print("O JOGO DUROU %d HORA(S)" % cont)

File: beginner/javascript/1046.js
var input = require('fs').readFileSync('stdin', 'utf8');

var valores = input.split(' ');

var horaIni = parseInt(valores.shift());
var horaFin = parseInt(valores.shift());
var cont = 0;

for(;;){
    horaIni++;
    cont++;
    if(horaIni == 24){
        horaIni = 0;
    }
    if(horaIni == horaFin){
        break;
    }
}

console.log("O JOGO DUROU " + cont + " HORA(S)");